sdk/eventhub/Azure.Messaging.EventHubs.Shared/tests/Core/ArgumentTests.cs

```diff
@@ -197,5 +197,39 @@ public void NotClosedEnforcesClosed()
             var target = "test";
             Assert.That(() => Argument.AssertNotClosed(true, target), Throws.InstanceOf<EventHubsException>().And.Property(nameof(EventHubsException.Reason)).EqualTo(EventHubsException.FailureReason.ClientClosed).And.Message.Contains(target));
         }
+
+        /// <summary>
+        ///   Verifies functionality of the <see cref="Argument.AssertWellFormedFullyQualifiedNamespace" /> method.
+        /// </summary>
+        ///
+        [Test]
+        [TestCase(null)]
+        [TestCase("")]
+        [TestCase("sb://myhub.servicebus.windows")]
```

Comment:
This was a semi-common mistake when copy/paste takes place from within the portal.  Because it was not validated within the constructors, it did not manifest until the first operation which required service communication.